app/view_models/workarea/storefront/google_product_feed_sku_view_model.rb

```diff
@@ -32,6 +49,23 @@ def inventory_sku
       def inventory_sku?
         options[:inventory_sku].present?
       end
+
+      private
+
+        def image
+          @image ||= sku_image || primary_image
+        end
+
+        def sku_image
```

Comment:
So what I was getting was that I think this method isn't necessary since you're using the `Storefront::ProductViewModel` now. Check out this code: https://github.com/workarea-commerce/workarea/blob/master/storefront/app/view_models/workarea/storefront/product_view_model/image_collection.rb#L39-L48 Since `options[:sku]` is being passed, that will find the SKU and match up the details with the image option for you.

Reply:
I can see why I was so confused before. I was looking at an ancient version of the file
`def primary
          @primary ||= browse_option_image ||
                         first ||
                         Catalog::ProductPlaceholderImage.cached
        end
`